src/pages/index.js

```diff
@@ -88,6 +100,7 @@ const cardTitleInput = addCardFormElement.querySelector(
 const cardUrlInput = addCardFormElement.querySelector(".modal__input_type_url");
 
 const cardSelector = "#card-template";
+
 /*----------------------------*/
 /*         Validation         */
 /*----------------------------*/
```

Comment:
From the 8th sprint all constants (`initialCards` and `settings`) should be in file `constants.js` in folder `utils` to clean up the main file.

Reply:
completed.